runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -99,10 +100,12 @@ export class PageControllerBase {
     ] = this.components.additionalValidationFunctions;
 
     this.saveAndContinueText = "Save and continue";
+    this.confirmAndContinueText = "Confirm and continue";
     this.continueText = "Continue";
 
     if (model?.def?.metadata?.isWelsh) {
```

Comment:
Can we have a util to do these conversions then once we have a initiative to move it to a i8n we just wanted to update that place ?